resources/config.edn

```diff
@@ -32,7 +32,7 @@
                                :staging :json
                                :prod :json}}
  :filestore #profile {:default {:local {:base-dir "/kixi-datastore"}
-                                :local-kinesis {:base-dir "/kixi-datastore"}
+                                #_:local-kinesis #_{:base-dir "/kixi-datastore"}
```

Comment:
?

Reply:
System only looks at the first key iirc, so the convention seems to be to comment out the other options.

Reply:
was just wondering at the change, don't make no difference.